test/util/util.go

```diff
@@ -1065,3 +1066,14 @@ func DeactivateWorkload(ctx context.Context, c client.Client, key client.ObjectK
 		g.Expect(c.Update(ctx, wl)).To(gomega.Succeed())
 	}, Timeout, Interval).Should(gomega.Succeed())
 }
+
+func SetNodeCondition(ctx context.Context, k8sClient client.Client, node *corev1.Node, conditionType corev1.NodeConditionType, conditionStatus corev1.ConditionStatus, time time.Time) {
```

Comment:
nit: I think for re-usability it is better to accept the entire NodeCondition here.